packages/node_modules/pouchdb-mapreduce-utils/src/index.js

```diff
@@ -1,6 +1,11 @@
 import argsarray from 'argsarray';
 import { nextTick } from 'pouchdb-utils';
 import { Set } from 'pouchdb-collections';
+import {
+  QueryParseError,
+  NotFoundError,
+  BuiltInError
+} from './errors';
```

Comment:
I moved these error objects into `pouchdb-mapreduce-utils` because they are needed by both `pouchdb-mapreduce` and `pouchdb-abstract-mapreduce`.